### PR TITLE
Feature/profile page iteration

### PR DESCRIPTION
**Pull request by FredrikAreschoug**
* Broke out sphere in layout file
* Added sphere to my home
* Added charity explanation bottom sheet

### CODE REVIEW

File: owldroid/src/main/res/layout/charity_how_dose_it_work_button.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (hedvigoscar) [inline owldroid/src/main/res/layout/charity_how_dose_it_work_button.xml]: filename => `charity_how_does_it_work_button.xml`

File: owldroid/src/main/res/layout/fragment_charity.xml [hunk 2]
```diff
@@ -85,6 +85,13 @@
 
                 </android.support.design.card.MaterialCardView>
 
+                <include
+                    android:id="@+id/charitySelectedHowDoseItWorkButton"
+                    layout="@layout/charity_how_dose_it_work_button"
```
Comment (hedvigoscar) [inline owldroid/src/main/res/layout/fragment_charity.xml]:
```suggestion
                    layout="@layout/charity_how_does_it_work_button"
```

File: owldroid/src/main/res/layout/sphere_container.xml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="@dimen/profile_sphere_side_length"
+    android:layout_height="@dimen/profile_sphere_side_length"
+    android:layout_marginTop="@dimen/base_margin_triple"
+    android:visibility="gone"
+    app:layout_constraintEnd_toEndOf="parent"
+    app:layout_constraintStart_toStartOf="parent"
+    app:layout_constraintTop_toTopOf="parent"
+    tools:visibility="visible"
+    tools:showIn="@layout/fragment_my_info">
```
Comment (hedvigoscar) [inline owldroid/src/main/res/layout/sphere_container.xml]: I like this! 🔥

File: owldroid/src/main/res/values/colors.xml [hunk 4]
```diff
@@ -18,4 +18,6 @@
     <color name="green">#1BE9B6</color>
     <color name="dark_green">#009175</color>
     <color name="pink">#FF8A80</color>
+    <color name="maroon">#AA0045</color>
+    <color name="bottom_sheet_dialog_title_background">#F9FAFC</color>
```
Comment (hedvigoscar) [inline owldroid/src/main/res/values/colors.xml]:
```suggestion
```

I think this is the same as `@color/off_white`

File: owldroid/src/main/res/drawable/bottom_sheet_dialog_fragment_background.xml [hunk 5]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+    <item>
+        <shape android:shape="rectangle">
+            <corners
+                android:topLeftRadius="@dimen/bottom_sheet_corner_radius"
+                android:topRightRadius="@dimen/bottom_sheet_corner_radius" />
+            <solid android:color="@color/bottom_sheet_dialog_title_background" />
```
Comment (hedvigoscar) [inline owldroid/src/main/res/drawable/bottom_sheet_dialog_fragment_background.xml]:
```suggestion
            <solid android:color="@color/off_white" />
```

File: owldroid/src/main/res/layout/sphere_container.xml (hunk 3 above)
Comment (FredrikAreschoug) [reply owldroid/src/main/res/layout/sphere_container.xml]: yeah it's neat :)

File: owldroid/src/main/res/layout/bottom_sheet_charity_explanation.xml [hunk 6]
```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:paddingStart="@dimen/base_margin_triple"
+    android:paddingEnd="@dimen/base_margin_triple"
+    android:paddingBottom="@dimen/base_margin_triple"
+    android:orientation="vertical">
+
```
[the remark this answers is not in the document]
Comment (FredrikAreschoug) [reply owldroid/src/main/res/layout/bottom_sheet_charity_explanation.xml]: :+1:

**Pull request merged**